src/pages/homePage/HomePage.jsx

```diff
@@ -1,30 +1,32 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect } from 'react';
 import { NavLink } from 'react-router-dom';
 import { useDispatch, useSelector } from 'react-redux';
-import { fetchDetails, showModal, updateCurrency } from '../../redux/features/detailsSlice';
+import {
+  fetchDetails, showModal, filter, updateMax, updateMin,
+} from '../../redux/features/detailsSlice';
 import CurrencyHome from '../../components/currencyCard/CurrencyCard';
 import Navbar from '../../components/navbar/Navbar';
 import './HomePage.css';
 
 const HomePage = () => {
   const dispatch = useDispatch();
```

Comment:
- [ ] As suggested by the previous reviewer, the homepage list of coins should be able to be filtered e.g. using a search bar using a search bar to allow users search for a particular coin or a dropdown to filter also.

Reply:
Hey @topeogunleye The settings Icon provides the dropdown

Reply:
> Hey @topeogunleye The settings Icon provides the dropdown

@Fene-87, please kindly submit for review again

Reply:
Hello @Fene-87 , I can see that you are dispatching filter on your homepage, and also see your search implementation through the price range 👍🏽